src/rules/convertRules.ts

```diff
@@ -75,6 +75,7 @@ export const convertRules = (
                         existingConversion.ruleArguments,
                         newConversion.ruleArguments,
                     ),
+                    notices: merger(existingConversion.notices, newConversion.notices),
```

Comment:
I'm not sure if there's ever a case where two rules will have different notices that need to be merged? There's a unit test failing on this branch that's related to this. Let's talk more about it and then I'll fix the test :)

Reply:
Oh that's a really good point I didn't think of. For now, can you just do `[...existingConversion.notices, ...newConversion.notices]`, and file a followup issue to investigate? I suspect we'll want to expand mergers into `merger.ruleArguments` and `merger.notices`, but agreed: +1 to talking more about it.

In the spirit of being agile, IMO we should get your changes in soon and do that as a followup.